src/mz-debug/README.md

```diff
@@ -1,3 +1,14 @@
 # `mz-debug`
 
 This tool allows us to debug a user's self-managed environment.
+
+## Run locally:
+To run locally, an example of a command is:
+
+```bash
+./bin/bazel run \
```

Comment:
Do we support running this tool not under bazel? If so should we provide an example here?